### PR TITLE
fixing linter errors

### PR DESCRIPTION
**Pull request by 09harsh**
# Description

Fixing linter errors

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Discussion

-

## Testing

Ran it manually

## Documentation
 -

## Follow-up

-

### CODE REVIEW

File: api/v1beta2/foundationdb_database_configuration.go [hunk 1]
```diff
@@ -76,9 +77,11 @@ type Region struct {
 // ExcludedServers represents the excluded servers in the database configuration
 type ExcludedServers struct {
 	// The Address of the excluded server.
+	// +kubebuilder:validation:MaxLength=20
```
Comment (johscheuer) [inline api/v1beta2/foundationdb_database_configuration.go:80]:
```suggestion
	// +kubebuilder:validation:MaxLength=48
```

The length. `20` is not enough for IPv6 e.g. `[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:65400'`

File: api/v1beta2/foundationdb_database_configuration.go [hunk 2]
```diff
@@ -76,9 +77,11 @@ type Region struct {
...
 	Address string `json:"address,omitempty"`
 
 	// The Locality of the excluded server.
+	// +kubebuilder:validation:MaxLength=100
```
Comment (johscheuer) [inline api/v1beta2/foundationdb_database_configuration.go:84]:
```suggestion
	// +kubebuilder:validation:MaxLength=200
```

W should add some space for longer localities or long values.

File: api/v1beta2/foundationdbcluster_types.go [hunk 3]
```diff
@@ -1414,10 +1414,12 @@ type ContainerOverrides struct {
 
 	// PeerVerificationRules provides the rules for what client certificates
 	// the process should accept.
+	// +kubebuilder:validation:MaxLength=3000
```
Comment (johscheuer) [inline api/v1beta2/foundationdbcluster_types.go:1417]: We should ensure that this length is good enough for our use cases.

File: api/v1beta2/image_config.go [hunk 4]
```diff
@@ -29,16 +29,20 @@ import "fmt"
 type ImageConfig struct {
 	// Version is the version of FoundationDB this policy applies to. If this is
 	// blank, the policy applies to all FDB versions.
+	// +kubebuilder:validation:MaxLength=20
 	Version string `json:"version,omitempty"`
 
 	// BaseImage specifies the part of the image before the tag.
+	// +kubebuilder:validation:MaxLength=20
```
Comment (johscheuer) [inline api/v1beta2/image_config.go:36]:
```suggestion
	// +kubebuilder:validation:MaxLength=200
```

The base image can include a registry.

File: api/v1beta2/image_config.go [hunk 5]
```diff
@@ -29,16 +29,20 @@ import "fmt"
...
 	BaseImage string `json:"baseImage,omitempty"`
 
 	// Tag specifies a full image tag.
+	// +kubebuilder:validation:MaxLength=20
```
Comment (johscheuer) [inline api/v1beta2/image_config.go:40]:
```suggestion
	// +kubebuilder:validation:MaxLength=100
```

**Review by johscheuer — CHANGES_REQUESTED** [review]

File: api/v1beta2/foundationdbcluster_types.go (hunk 3 above)
Comment (09harsh) [reply api/v1beta2/foundationdbcluster_types.go:1417]: I checked this on one of our QA cluster. The value was something around 2600. So I put 3k here.

**Review by johscheuer — APPROVED** [review]

**Pull request merged**